Source/Mosa.Kernel.BareMetal/Korlib/EnvironmentPlug.cs

```diff
@@ -9,21 +11,23 @@ public class EnvironmentPlug
 	[Plug("System.Environment::FailFast")]
 	public static void FailFast(string message)
 	{
-		Debug.WriteLine("DEBUG: ");
-		Debug.WriteLine(message);
+		var deviceService = Kernel.ServiceManager.GetFirstService<DeviceService>();
+		var graphicsDevices = deviceService.GetDevices<IGraphicsDevice>(DeviceStatus.Online);
+
+		foreach (var device in graphicsDevices) ((IGraphicsDevice)device.DeviceDriver).Disable();
 
+		Debug.WriteLine("*** FailFast ***");
+		Debug.WriteLine(message);
 		Debug.Fatal();
 	}
 
 	[Plug("System.Environment::Exit")]
 	public static void Exit(int exitCode)
 	{
-		Debug.Fatal();
```

Comment:
Keep as is.

Let's not combine Environment with Debug.

Reply:
It makes sense to use PCService in Environment.Exit(). In regular .NET, this would exist the application, so it would only make sense to shut down the machine here.